### PR TITLE
#33 migrate to inquirer

### PR DESCRIPTION
**Pull request by lvenier**

### CODE REVIEW

**Comment by lvenier** [comment]
@martinlevesque : might require few adjustement as inquirer handle input validation in a different way.

File: modules/auth.js [hunk 1]
```diff
@@ -21,95 +20,89 @@ function authenticate (email, password) {
...
 }
 
-function signupApi (email, password, password_confirmation, newsletter) { // eslint-disable-line
+function signupApi (email, password, passwordConfirmation, newsletter) {
   return apiRequest.post('account/register', {
     account: {
       email,
       password,
-      password_confirmation,
+      passwordConfirmation,
```
Comment (martinlevesque) [inline modules/auth.js:28]: should be changed to:

`password_confirmation: passwordConfirmation,`

the variable name is important

Comment (lvenier) [reply modules/auth.js:28]: not sure I got it right. you mean the variable name ?
I might have missed something as the var name could be anything ?

File: modules/auth.js [hunk 2]
```diff
@@ -21,95 +20,89 @@ function authenticate (email, password) {
...
+async function wantsNewsletter () {
+  const schema = [{
+    type: 'input',
+    message: 'Subscribe to the newsletter ([y]es or [n]o) ?',
+    name: 'wantsNewsletter',
+    choices: ['y', 'n'],
+    default: 'y'
+  }]
+  const result = await inquirer.prompt(schema)
   return new Promise((resolve, reject) => {
```
Comment (martinlevesque) [inline modules/auth.js:67]: replace this return by (and remove the new Promise(...)):

`return result.wantsNewsletter === 'y' ? 1 : 0`

**Comment by martinlevesque** [comment]
thanks for your contribution, the UI is good.
Made 2 comments, 2 bugs to fix.

File: modules/auth.js (hunk 1 above)
Comment (martinlevesque) [reply modules/auth.js:28]: It's currently calling the api with to register an account with:
```
{
account: {
 ...
passwordConfirmation,
...
}
}
```

but the API looks for:

```
{
account: {
 ...
password_confirmation,
...
}
}
```

**Comment by lvenier** [comment]
@martinlevesque : ok got it. PR updated.

**Pull request merged**